### PR TITLE
Liftr-NewRelic Resubscribe API - 2024-10-01 stable version

### PR DESCRIPTION
**Pull request by shalmolineogi**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

Adding a new api spec for supporting resubscribe feature

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (mikeharder) [inline .devcontainer/devcontainer.json:1]: Please revert all changes to files outside the spec you are modifying

**Review by mikeharder — DISMISSED** [review]
Revert all changes to files outside the spec you are modifying

**Comment by shalmolineogi** [comment]
> Revert all changes to files outside the spec you are modifying

Reverted

**Review by dipeshbhakat-microsoft — APPROVED** [review]
approved

**Comment by dipeshbhakat-microsoft** [comment]
> @shalmolineogi please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Microsoft"

**Comment by shalmolineogi** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**